Assets/ScriptsWithDependencies/RegistrationManager.cs

```diff
@@ -91,7 +101,28 @@ public void VerifyUsernameOnValueChanged(string usernameInput)
         }
         else
         {
-            InfoText.text = "Benutzername muss zwischen 5-20 Zeichen lang sein";
+            InfoText.text = "Benutzername muss zwischen 3-20 Zeichen lang sein";
+            SubmitButton.interactable = false;
+        }
+    }
+
+    /// <summary>
+    /// called after user tipped in the first password field.
+    /// If the given password is valid, it will be compared to the second one
+    /// which confirms whether the user entered the same password twice
+    /// </summary>
+    /// <param name="passwordInput">first input field for password</param>
+    public void CheckPasswordsAreSameAfterTippedOnFirstPassField(string passwordInput)
+    {
+        IsPasswordConfirmed = PasswordConfirmField.text == passwordInput;
+        if (IsPasswordConfirmed)
+        {
+            InfoText.text = "";
+            VerifyAll();

```

Comment:
wenn du hier ein return statement machst, brauchst du keine else verzweigung. Sieht generell lesbarer aus. Kannst du auch direkt auf resolved setzen, aber würde ich bevorziehen zu sehen, falls möglich! :)
```suggestion
            VerifyAll();
            return;
            }
        InfoText.text = "Passwörter sind nicht identisch";
        SubmitButton.interactable = false;
```

Reply:
done